content/3_Silver/Two_Pointers.mdx

```diff
@@ -280,3 +280,72 @@ public class Books {
 ## Problems
 
 <Problems problems="ad" />
+
+## Quiz
+
+<Quiz>
+  <Quiz.Question>
+    What is the two pointers algorithm?
+    <Quiz.Answer correct>
+      Keeping track of two pointers to find a pair of indices that satisfy some condition.
+      <Quiz.Explanation>
+        Correct.
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      A way to answer range sum queries in $\mathcal{O}(1)$ time.
+      <Quiz.Explanation>
+        Incorrect. You might be thinking of [Prefix Sums](https://usaco.guide/silver/prefix-sums), another silver topic.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      A divide and conquer algorithm that splits an array into two halves to efficiently find an element satisfying a certain condition.
+      <Quiz.Explanation>
+        Incorrect.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    When would a solution be impossible when doing a collision two pointers algorithm? (two pointers move from the ends of the array to the middle). 
```

Comment:
I don't get this question. Can you specifically state the problem that this solution is for?

Reply:
there isn't a specific problem, but an example could be [Sum of Three Values](https://cses.fi/problemset/task/1641). The idea was that this specific case of two pointers was if the two pointers were at opposite ends of the array.

Reply:
Can you specifically state that then? Probably a better way to format the question would be to have pseudocode and leave the condition in the while loop empty.

Reply:
also add what the problem is? If it was "find two (not necessarily distinct) numbers that sum to the target, then the answer would be different.

Reply:
oops, adding now